src/component/inspect-request/request-headers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const {object} = React.PropTypes;
+
+const RequestHeaders = (props) => {
```

Comment:
Yeah, `KeyValue` did hurt a bit. At the time I named this, this was before you added the query params feature, and then I discovered it upon rebasing. I kind of justified it to myself by saying you could *maybe* consider query parameters to be a request header. It's a bit flimsy logic, but it kinda works.

Maybe `RequestDetails`? I'm open to suggestions. Won't hurt to rename later either.